invenio_communities/models.py

```diff
@@ -411,10 +415,23 @@ def oaiset_spec(self):
             community_id=self.id)
 
     @property
+    def oaiset(self):
+        """Return the corresponding OAISet for given community.
+
+        :returns: returns OAISet object corresponding to this community.
+        :rtype: `invenio_oaiserver.models.OAISet`
+        """
+        if current_app.config["COMMUNITIES_OAI_ENABLED"]:
+            from invenio_oaiserver.models import OAISet
+            return OAISet.query.filter_by(spec=self.oaiset_spec).one()
+        else:
+            raise Exception("OAI is disabled for InvenioCommunities.")
```

Comment:
Hmm, I like the idea but since it's not clear where to inject this (outside model.py would be confusing) I'll push with ``None`` as return value.